Shippo/APIResource.cs

```diff
@@ -433,6 +438,49 @@ public ShippoCollection<Manifest> AllManifests (Hashtable parameters)
 
         #endregion
 
+        #region Batch
+
+        public Batch CreateBatch (String carrierAccount, String servicelevelToken, String labelFiletype,
```

Comment:
If possible, please have enum for `labelFileType` and similarly for `objectStatus` in Batch class.